packages/util/src/helpers.ts

```diff
@@ -1,5 +1,21 @@
 import { isHexString } from 'ethjs-util'
 
+/**
+ * Checks if a number string is a hex string, if not, converts and returns it 
+ * @param {string} input string to check, convert, and return
+ */
+ export const toHex = function (input?: string) {
+  if (!input) return undefined
+  if (!isHexString(input)) {
+    try {
+      return '0x' + parseInt(input, 10).toString(16)
+    } catch {
+      const msg = `This method only supports number or 0x-prefixed hex strings but input was: ${input}`
+      throw new Error(msg)
+    }
+  }
```

Comment:
this is just personal preference for less nesting

```suggestion
  if (isHexString(input)) return input
  try {
    return '0x' + parseInt(input, 10).toString(16)
  } catch {
    const msg = `This method only supports number or 0x-prefixed hex strings but input was: ${input}`
    throw new Error(msg)
  }
```